scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -1096,6 +1096,8 @@ def run(self):
             #
             if isinstance(harness, Pytest):
                 harness.pytest_run(instance.handler.get_test_timeout())
+            elif isinstance(harness, Robotframework):
+                harness.robot_run(instance.handler.get_test_timeout())
```

Comment:
maybe better to check if it is `TwisterLibraryHarness` and call common method `run` (both `pytest_run` and `robot_run` are the same